### PR TITLE
Docs: Update Data Exports

### PR DESCRIPTION
**Pull request by ChrisHuynh333**
## What does this PR do and why?
This PR updates the Data Export docs to include Linelist exports

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

## How to set up and validate locally
1. Build the docs
```
cd docs-site
pnpm update
pnpm build
npm run serve
```
2. Read and verify the docs are accurate (Use IRIDA Next > Export your data)

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

- [x] I have evaluated the [PR acceptance checklist](https://phac-nml.github.io/irida-next/docs/development/development_processes/code_review#acceptance-checklist) for this PR.

### CODE REVIEW

File: docs-site/docs/user/export/working-with-exports.md [hunk 1]
```diff
@@ -32,7 +32,29 @@ To create a sample export:
   2. From the left sidebar, select **Samples**
   3. Click the checkbox of each sample you'd like to export
   4. Click **Create Export**
-  5. A pop-up will appear asking if you'd like to give the export a name and whether you'd like to receive an e-mail notification when the export is ready to download. Neither of these are required to create the export.
+  5. In the dropdown, select **Sample Export**
+  6. A pop-up will appear asking the following:
+  * The file formats you'd like to include within the export. This allows you to filter out any file formats you do not wish to export. You are required to include at least one file format to create an export.
```
[the remark this answers is not in the document]
Comment (ChrisHuynh333) [reply docs-site/docs/user/export/working-with-exports.md]: Updated in 433511d0149c63150c94812f9f4d0d396741fa2b

File: docs-site/docs/user/export/getting-started.md [hunk 2]
```diff
@@ -21,6 +22,9 @@ The following is required to create an export:
 * Sample Export
   * Project that has at least one associated sample, and files uploaded to the sample.
   * At least the Analyst role for the Project or Group containing the samples for export.
+* Linelist Export
+  * Project that has at least one associated sample that contains metadata.
+  * At least the Analyst role for the Project or Group containing the samples for export.
 * Analysis Export
   * User Workflow Execution
     * A user workflow execution in the completed state
```
[the remark this answers is not in the document]
Comment (ChrisHuynh333) [reply docs-site/docs/user/export/getting-started.md]: Updated in 433511d0149c63150c94812f9f4d0d396741fa2b

File: docs-site/docs/user/export/working-with-exports.md [hunk 3]
```diff
@@ -28,11 +28,33 @@ Prerequisites:
...
+   - If you'd like to give the export a name (not required).
+   - If you'd like to receive an e-mail notification when the export is ready to download (not required).
+
+## Create Linelist Export
+
+Prerequisites:
+  * A project must contain at least one sample, and that sample must contain metadata.
+  * You must have at least the Analyst role to the project and/or group you will create the export from.
+
+To create a sample export:
```
Comment (deepsidhu85) [inline docs-site/docs/user/export/working-with-exports.md:47]: This should read `To create a sample linelist export:`

**Review by deepsidhu85 — CHANGES_REQUESTED** [review]
This looks good to me. Just have one change below

File: docs-site/docs/user/export/working-with-exports.md (hunk 3 above)
Comment (ChrisHuynh333) [reply docs-site/docs/user/export/working-with-exports.md]: Thanks, fixed in ac1ecd63fabba137949d87474056d7d98bf6d518

**Review by deepsidhu85 — APPROVED** [review]
Looks good to me!

**Pull request merged**